examples/layer-browser/src/index.js

```diff
@@ -0,0 +1,41 @@
+/* global document */
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import App from './app';
+import FPSStats from 'react-stats-zavatta';
+
+class Root extends Component {
+  constructor(props) {
+    super(props);
+    this._appState = null;
```

Comment:
I think you can initialize `_appState` as a static property (if the babel config there is permits), not sure how much we want to use these kind of features though

Reply:
It does not